src/Angor/Avalonia/AngorApp/Sections/Wallet/BroadcastedTransactionDesign.cs

```diff
@@ -0,0 +1,17 @@
+using System.Threading.Tasks;
+using AngorApp.Model;
+using CSharpFunctionalExtensions;
+
+namespace AngorApp.Sections.Wallet;
+
+public class BroadcastedTransactionDesign : IBroadcastedTransaction
+{
+    public string Address { get; init; }
+    public decimal FeeRate { get; set; }
+    public decimal TotalFee { get; set; }
+    public decimal Amount { get; init; }
```

Comment:
One thing that we have discuss earlier is that perhaps we should always represent amounts as sats and never as Bitcoin (decimal) only when we display we make the conversion